packages/desktop-gui/cypress/integration/project_nav_spec.js

```diff
@@ -330,13 +337,13 @@ describe('Project Nav', function () {
 
       it('displays local storage browser name in chosen', () => {
         cy.get('.browsers-list .dropdown-chosen')
-        .should('contain', 'Chromium')
+        .should('contain', 'Canary').and('not.contain', 'Edge')
```

Comment:
ok, so what happens if the user already has `chromium` in the local storage and then this version tries to parse it as JSON?

Reply:
https://github.com/cypress-io/cypress/blob/80799a12067060a68873c0a6c37c203be2a6cf95/packages/desktop-gui/src/project/project-model.js#L258-L263